keys_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestClient_CreateKey(t *testing.T) {
+func TestClient_CreateAuthKey(t *testing.T) {
```

Comment:
Since you chose to implement `CreateAuthKey` separately from `CreateKey`, for full test coverage, you would want a test for both methods. If you take my suggestion of having `CreateKey` delegate to `CreateAuthKey`, you can keep the existing test with just small modifications.